bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -317,7 +299,7 @@
 			<channel id="icon-id" typeId="condition-icon-id"/>
 			<channel id="temperature" typeId="system.outdoor-temperature"/>
 			<channel id="apparent-temperature" typeId="apparent-temperature"/>
-			<channel id="pressure" typeId="system.barometric-pressure"/>
+			<channel id="pressure" typeId="systembarometric-pressure"/>
```

Comment:
This change looks unintended?

Reply:
Good catch, thanks!
I've reverted that change in https://github.com/openhab/openhab-addons/pull/16369/commits/1d57122ed94740d21e0075faf3bed9e400ab6e38.